tests/unit/swagger/schemaDetails.spec.ts

```diff
@@ -39,3 +39,30 @@ describe('Schema details generation', () => {
 
   it('should set API license if provided', () => expect(licenseName).to.equal(getDefaultOptions().license));
 });
+
+describe('Custom Attribute schema generation', () => {
+  const metadata = new MetadataGenerator('./tests/fixtures/controllers/methodController').Generate();
+  const spec = new SpecGenerator(metadata, getDefaultOptions()).GetSpec();
```

Comment:
```suggestion
  const spec = new SpecGenerator2(metadata, getDefaultOptions()).GetSpec();
```
Also, add tests for SG3 please.